lib/ValueType.pm6

```diff
@@ -1,9 +1,15 @@
 use v6.d;
 
+# no need for :ver or :auth as it is hidden to the outside world
+my role ValueType::Excluded {}
+multi trait_mod:<is> (Attribute \attr, :$excluded-from-valuetype!) is export {
+    attr does ValueType::Excluded
+}
+
 role ValueType:ver<0.0.3>:auth<cpan:ELIZABETH> {
-    has $!WHICH;
+    has $!WHICH is excluded-from-valuetype;
 
-    my @attributes = ::?CLASS.^attributes.grep: *.name ne 'WHICH';
+    my @attributes = ::?CLASS.^attributes.grep: { $_ !~~ ValueType::Excluded };
```

Comment:
I seem to recall that `!(foo ~~ bar)` performs orders of magnitude better than `foo !~~ bar`.  Am I remembering that correctly?

Reply:
I don't think it matters much in this case.

It will only be run during compilation of a module that `does ValueType`.
I would think that objects that are ValueTypes will have relatively few attributes.

Mainly I wanted to find out if this was a feature that you would decide to accept.
There is a reason I didn't bother writing in the changelog, or the documentation.

Reply:
True.

I'm willing to accept such a feature.  I'd also be welcome to you becoming maintainer for it  :-)

Reply:
About the only problem with me being a maintainer of anything is that I haven't kept on top of things like my github notifications recently. Although I did get emails about this pull request, so it may not be that big of a problem.

Other than that, I would have no problem with becoming the maintainer for anything.